package.json

```diff
@@ -39,9 +39,9 @@
     "lint-fix": "prettier client-src/js-src client-src/elements --write && eslint \"client-src/js-src/**/*.{js,ts}\" \"client-src/elements/**/*.{js,ts}\" --fix",
     "presubmit": "npm test && npm run webtest && npm run lint && npm run mypy",
     "pylint": "pylint --output-format=parseable *py api/*py framework/*py internals/*py pages/*py",
-    "staging": "npm run build && ./scripts/deploy_site.sh `git describe --always --abbrev=7 --match 'NOT A TAG' --dirty='-tainted'` cr-status-staging",
-    "staging-rc": "npm run build && ./scripts/deploy_site.sh rc cr-status-staging",
-    "deploy": "npm run build && ./scripts/deploy_site.sh `git describe --always --abbrev=7 --match 'NOT A TAG' --dirty='-tainted'`",
+    "staging": "npm run clean-setup && npm run build && ./scripts/deploy_site.sh `git describe --always --abbrev=7 --match 'NOT A TAG' --dirty='-tainted'` cr-status-staging",
+    "staging-rc": "npm run clean-setup && npm run build && ./scripts/deploy_site.sh rc cr-status-staging",
+    "deploy": "npm run clean-setup && npm run build && ./scripts/deploy_site.sh `git describe --always --abbrev=7 --match 'NOT A TAG' --dirty='-tainted'`",
     "deploy-rc": "npm run build && ./scripts/deploy_site.sh rc",
```

Comment:
You should add it to the deploy-rc target also.  We haven't been using the *-rc targets, but you might as well keep it in step with the others.

Reply:
Ah missed that one. Good catch!